run_marqo.sh

```diff
@@ -45,52 +61,70 @@ if [[ ! $OPENSEARCH_URL ]]; then
       wait "$setup_dind_pid"
   fi
 
-  echo "Starting supervisor"
+  if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
+    echo "Starting supervisor"
+  fi
+
   /usr/bin/supervisord -n >> /dev/null 2>&1 &
 
-  dockerd &
-  echo "called dockerd command"
+  dockerd > /dev/null 2>&1 &
+
+  if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
+    echo "Called dockerd command. Waiting for dockerd to start."
+  fi
 
-  echo "Waiting for processes to be running"
   processes=(dockerd)
   for process in "${processes[@]}"; do
       wait_for_process "$process"
       if [ $? -ne 0 ]; then
           echo "$process is not running after max time"
           exit 1
       else
-          echo "$process is running"
+          if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
+            echo "$process is running"
+          fi
       fi
   done
   OPENSEARCH_URL="https://localhost:9200"
   OPENSEARCH_IS_INTERNAL=True
   if [[ $(docker ps -a | grep marqo-os) ]]; then
       if [[ $(docker ps -a | grep marqo-os | grep -v Up) ]]; then
-        docker start marqo-os &
+        docker start marqo-os > /dev/null 2>&1 &
         until [[ $(curl -v --silent --insecure $OPENSEARCH_URL 2>&1 | grep Unauthorized) ]]; do
           sleep 0.1;
         done;
-        echo "Opensearch started"
+        echo "Marqo-OS started"
       fi
-      echo "OpenSearch is running"
+      echo "Marqo-OS is running"
   else
-      echo "OpenSearch not found; running OpenSearch"
-      docker run --name marqo-os -id -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3 &
-      docker start marqo-os &
+      echo "Marqo-OS not found; starting Marqo-OS..."
+      if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
+        docker run --name marqo-os -id -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3 &
+      else
+        docker run --name marqo-os -id -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" marqoai/marqo-os:0.0.3  > /dev/null 2>&1 &
+      fi
+      if [ "$MARQO_LOG_LEVEL" = "debug" ]; then
+        docker start marqo-os > /dev/null 2>&1 &
+      fi
       until [[ $(curl -v --silent --insecure $OPENSEARCH_URL 2>&1 | grep Unauthorized) ]]; do
         sleep 0.1;
       done;
+      echo "Marqo-OS started successfully."
   fi
+else
+    echo "Found Marqo-OS URL. Skipping internal Marqo-OS configuration"
```

Comment:
nit: add period to the end of the sentence for uniformity.

Reply:
done